### PR TITLE
Add basic infos about Gamify-IT

### PR DESCRIPTION
**Pull request by cdingm**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1 +1,21 @@
 # Gamify-IT Quickstart Guide
```
Comment (delvh) [inline profile/README.md]:
```suggestion
# Gamify-IT
```

File: profile/README.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+
+Gamify-IT is a minigame platform for students. With Gamification the students are able to learn lecture content through variable minigames. There is a world where the student can walk through and interact with.
```
Comment (delvh) [inline profile/README.md]:
```suggestion
_Gamify-IT_ is a minigame platform for students.  
Students are able to learn content by playing minigames.  
Additionally, they can explore the large overworld and with interact it.  
```

File: profile/README.md [hunk 3]
```diff
@@ -1 +1,33 @@
 # Gamify-IT Quickstart Guide
+
+Gamify-IT is a minigame platform for students. With Gamification the students are able to learn lecture content through variable minigames. There is a world where the student can walk through and interact with.  
+A lecturer sets up the World and the minigames. He is able to add Questions about lecture content to the minigames, add hints for the students, etc.  
+There is a progress system for the students to unlock new content.
```
Comment (delvh) [inline profile/README.md]:
```suggestion
A lecturer configures the world and minigames.  
They determine what questions the students are asked in a minigame, what hints to give the students (i.e. when a student talks to an NPC), … 
```

File: profile/README.md [hunk 4]
```diff
@@ -1 +1,33 @@
...
+
+## Overview
+
+### world
```
Comment (delvh) [inline profile/README.md]:
```suggestion
### World
```

File: profile/README.md [hunk 5]
```diff
@@ -1 +1,33 @@
...
+
+![World](https://raw.githubusercontent.com/Gamify-IT/docs/main/images/overworld.webp)
+The Player (in the middle of the picture) is able to move around the World and interact with variable content.
```
Comment (delvh) [inline profile/README.md]:
```suggestion
The player (in the middle of the picture) is able to move around the World and interact with customizable content.
```

File: profile/README.md [hunk 6]
```diff
@@ -1 +1,33 @@
...
+
+### Lecturer Interface
+
+![Lecturer Interface](https://raw.githubusercontent.com/Gamify-IT/docs/main/images/lecturer-interface.webp)
+The Lecturer has access to an interface where he can configure the worlds (for example add questions regarding the topic of the lecture)
```
Comment (delvh) [inline profile/README.md]:
```suggestion
The lecturer can configure everything with the [lecturer interface](https://github.com/Gamify-IT/lecturer-interface).  
In the future, they might also see course statistics there.
```

File: profile/README.md [hunk 7]
```diff
@@ -1 +1,33 @@
...
+
+## Repositories
+
+information about all the different repositories
+
+### [Docs](https://github.com/Gamify-IT/docs)
+
+The Docs repository contains the documentation about the minigame platform
```
Comment (delvh) [inline profile/README.md]:
```suggestion
The [docs repo](https://github.com/Gamify-IT/docs) contains the complete documentation.  
We recommend <https://gamifyit-docs.readthedocs.io/en/latest/> to view the docs as it always displays the content in a visually enhanced way.
```

File: profile/README.md [hunk 8]
```diff
@@ -1 +1,33 @@
...
+### Lecturer Interface
+
+![Lecturer Interface](https://raw.githubusercontent.com/Gamify-IT/docs/main/images/lecturer-interface.webp)
+The Lecturer has access to an interface where he can configure the worlds (for example add questions regarding the topic of the lecture)
+
+## Repositories
+
+information about all the different repositories
+
+### [Docs](https://github.com/Gamify-IT/docs)
```
Comment (delvh) [inline profile/README.md]:
```suggestion
### Docs
```

File: profile/README.md [hunk 9]
```diff
@@ -1 +1,33 @@
...
+
+The Docs repository contains the documentation about the minigame platform
+
+### [Overworld](https://github.com/Gamify-IT/overworld)
+
+The Overworld repository contains the Unity Project on which the World is build with. It contains all the Assets, Scripts, Game Logic, etc. that is needed for the World to run.
```
Comment (delvh) [inline profile/README.md]:
```suggestion
### Overworld

The [overworld repo](https://github.com/Gamify-IT/overworld) contains all the assets, scripts, game logic, … to display and walk through the world.
```

File: profile/README.md [hunk 10]
```diff
@@ -1 +1,33 @@
...
+
+### [Overworld-Backend](https://github.com/Gamify-IT/overworld-backend)
+
+The Overworld-Backend repository contains the Java backend of the World. All relevant data of the World is stored in the Overworld-Backend
```
Comment (delvh) [inline profile/README.md]:
```suggestion
### Overworld Backend

The [overworld backend repo](https://github.com/Gamify-IT/overworld-backend) stores all overworld data, whether it be the content of a course or player statistics.  
Minigame runs are stored in the backend of the respective minigame.
```

**Comment by cdingm** [comment]
i would say we review this and then merge, then we can add more parts of the readme

File: profile/README.md [hunk 11]
```diff
@@ -1 +1,117 @@
...
+
+The [issues repo](https://github.com/Gamify-IT/issues) contains all the issues across the Project. Instead of opening issues in the individual repos, they are created here.
+
+### Landing Page
+
+The [landing page repo](https://github.com/Gamify-IT/landing-page)
+
+### Run config
+
+The [run config repo](https://github.com/Gamify-IT/run-config)
```
Comment (delvh) [inline profile/README.md:109]: :laughing: I'll accept it for now.

**Review by delvh — APPROVED** [review]

**Review by Gr33ndev — APPROVED** [review]

**Pull request merged**